pkg/kubelet/volumes.go

```diff
@@ -243,7 +259,7 @@ func (kl *Kubelet) getPodVolumesFromDisk() map[string]cleanerTuple {
 				continue
 			}
 			if unmounter == nil {
-				glog.Errorf("Could not create volume unmounter for %s: %v", volume.Name, errUnsupportedVolumeType)
+				glog.Errorf("Could not create volume unmounter for %s: %v", volume.Name, createVolumeTypeError(volume.Name))
```

Comment:
We should not need to check for a nil unmounter.  Never return (nil, nil) from a method like this.